arctic/store/_pandas_ndarray_store.py

```diff
@@ -195,6 +196,17 @@ def read(self, arctic_lib, version, symbol, read_preference=None, date_range=Non
             item = self._daterange(item, date_range)
         return item
 
+    def get_data_info(self, version):
+        """
+        parses out the relevant information in version
+        and returns it to the user in a dictionary
+        """
+        ret = {}
+        ret['dtype'] = ast.literal_eval(version['dtype'])
```

Comment:
is there a change you'd like to me make? Wasn't sure the best way to convert a string of a list of tuples to a list of tuples.